cmd/config/subcommand/sandbox/sandbox_config.go

```diff
@@ -0,0 +1,14 @@
+package sandbox
+
+//go:generate pflags Config --default-var DefaultConfig
+var (
+	DefaultConfig = &Config{
+		Debug: false,
```

Comment:
Lets keep the default as true , so that the user knows whats happening instead of being impatient and waiting.